app-misc/pet/pet-0.2.4.ebuild

```diff
@@ -35,23 +33,29 @@ KEYWORDS="~amd64"
 IUSE="zsh-completion"
 
 # dev-go/toml doesn't provide sources
-DEPEND="dev-go/go-crypto
-	dev-go/go-net
-	dev-go/go-oauth2
-	dev-go/go-protobuf
-	dev-go/go-sys"
+DEPEND="dev-lang/go:=
+	dev-go/go-crypto:=
+	dev-go/go-net:=
+	dev-go/go-oauth2:=
+	dev-go/go-protobuf:=
+	dev-go/go-sys:="
 RDEPEND="${DEPEND}
 	zsh-completion? ( app-shells/zsh-completions )"
 
+DOC_CONTENTS="${CATEGORY}/${PN}: You should consider to
```

Comment:
1. Portage already puts `cat/pn` above the message after finishing the builds.
2. To be honest, I don't think think a single sentence is really a reason to create a `README.gentoo` file.
3. Instead, you could use `has_version` to display it only when the dependency is not installed.
4. Also, there's `optfeature` thing in eutils that is kinda relevant, yet it's your choice if you want to use it.